ui/orbital.c

```diff
@@ -686,17 +688,17 @@ static void* orbital_display_main(void* arg)
                 }
             }
         }
-		
-        if (ui.minimized == false) {
-            // Frame
+
+        if (ui.window_state == maximized) {
```

Comment:
I think rendering frames shouldn't be conditionally triggered *only* when the window is maximized. The only real reason to disable rendering is if the window is not visible, i.e. if it's minimized `ui.window_state != minimized`. But as I mentioned in my comment above, I see no reason to keep such a verbose `window_state` either.

Reply:
The way i thought when doing this was it would either be one or the other, minimized or maximized, shown or hidden?

But yeah i agree there really isn't much need for window_state at the moment